tensorflow/lite/micro/testing/test_utils.h

```diff
@@ -29,6 +29,9 @@ limitations under the License.
 namespace tflite {
 namespace testing {
 
+// Float tolerance for quantized int16 tests
+constexpr float kQuantizedToleranceInt16 = 2 * (1. / 4096);
```

Comment:
Can you explain the reasoning behind this value in the comment?

Reply:
It was copied from other int16 tests but you are right to point out that it's arbitrary and I noticed this tolerance was a bit too relaxed on some output values. I changed it to relative tolerance instead as I think it's more robust and meaningful.